### PR TITLE
Make VR lines and hole highlighting optional

### LINKED ISSUE #320: Hide VR helper lines by default
As discussed before - I'll add a checkbox for that. Currently these lines a tad too obtrusive and make it impossible to scroll iframes (eg. see toplayer).

### PR DESCRIPTION
**Pull request by kdzwinel**
Fixes #320 

screen shot 2015-09-21 at 17 20 26

### CODE REVIEW

File: src/docs/_layouts/default.html [hunk 1]
```diff
@@ -21,6 +21,43 @@
...
+
+    function toggleHoles(value) {
+        if(value) {
+            document.body.classList.add('show-holes');
+        } else {
+            document.body.classList.remove('show-holes');
+        }
+
+        showHolesCheckbox.checked = value;
+        localStorage.showHoles = value;
```
Comment (kdzwinel) [inline src/docs/_layouts/default.html]: settings 'stick'

**Comment by matzimowski** [comment]
with love :+1:

File: src/docs/_layouts/default.html [hunk 2]
```diff
@@ -21,6 +21,43 @@
...
+        } else {
+            document.body.classList.remove('show-vr-lines');
+        }
+
+        showVRLinesCheckbox.checked = value;
+        localStorage.showVRLines = value;
+    }
+
+    showHolesCheckbox.addEventListener('change', function () {
+        toggleHoles(this.checked);
```
Comment (aju) [inline src/docs/_layouts/default.html]: Do we need it?

File: src/docs/_sass/_navigation.scss [hunk 3]
```diff
@@ -76,14 +76,22 @@
     }
   }
 
-  .version {
+  aside {
```
Comment (aju) [inline src/docs/_sass/_navigation.scss]: We should fix it one day to BEM :(

**Comment by kdzwinel** [comment]
@aju I cleaned up the code a bit - much shorter now!

**Comment by aju** [comment]
:OK:

**Pull request merged**